cmd/runtimetest/main.go

```diff
@@ -594,6 +595,20 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMountLabel(spec *rspec.Spec) error {
+	for _, mount := range spec.Mounts {
+		fileLabel, err := label.GetFileLabel(mount.Destination)
+		if err != nil {
+			return fmt.Errorf("Failed to get mountLabel of %v", mount.Destination)
+		}
+		if fileLabel != spec.Linux.MountLabel {
+			return fmt.Errorf("Expected mountLabel %v, actual %v", spec.Linux.MountLabel, fileLabel)
```

Comment:
There are no [RFC 2119 requirements for the runtime for `linux.mountLabel`][1].  I think we need to get spec language about what the runtime MUST do before we can fail a runtime for not doing it (opencontainers/runtime-spec#746).

[1]: https://github.com/opencontainers/runtime-spec/blob/fae0d09a24b78ef17e43d411583780b9f2a48d5b/config-linux.md#mount-label